rpc/frontend/src/main/scala/io/udash/rpc/DefaultServerRPC.scala

```diff
@@ -26,7 +26,7 @@ object DefaultServerRPC {
             ServerRPCType : DefaultServerUdashRPCFramework.AsRealRPC]
            (localRpc: ClientRPCType, serverUrl: String = "/atm/",
             exceptionsRegistry: ExceptionCodecRegistry = new DefaultExceptionCodecRegistry,
-            rpcFailureInterceptors: Seq[(Throwable) => Unit]): ServerRPCType = {
+            rpcFailureInterceptors: Seq[PartialFunction[Throwable, Any]] = Seq.empty): ServerRPCType = {
```

Comment:
You are using these `PartialFunction`s with `.foreach`, which calls `.apply` and fails when given `Throwable` type is not handled. I'd expect from this method signature that I can intercept specific exception types. I believe only `.collect` checks `.isDefinedAt` before applying the function.